test/unitTests/fixtures/server-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
This will probably cause an issue with secret scanning -- can this be auto generated as part of the build so it doesn't need to be checked in? or is this required to send it somewhere.